docs/tutorials/cups/index.md

```diff
@@ -0,0 +1,60 @@
+# Control and User Plane (CUPS) Separation
+
+In this tutorial, we will deploy and run the SD-Core 5G core network using Juju. We will also deploy a radio and cellphone simulator to simulate usage of this network.
+
+```{image} ../../images/sdcore_cups_network_diagram.png
+:alt: Network diagram
+:align: center
+```
+
+To complete this tutorial, you will need four machines (virtual or physical), with access to the networks as follows.
+
+| Machine | CPUs | RAM | Disk | Networks |
+| ------- | ---- | --- | ---- | -------- |
+| Control Plane Kubernetes Cluster     | 4 | 8g | 40g | `management` |
+| User Plane Kubernetes Cluster        | 4 | 4g | 20g | `management`, `access`, `core` |
+| Juju Controller + Kubernetes Cluster | 2 | 4g | 40g | `management` |
+| gNB Simulator Kubernetes  Cluster    | 4 | 4g | 20g | `management`, `ran` |
+
+
+## Sample Values
+
+For the purpose of this guide, the following values will be used.
+
+### Networks
+
+| Name | Subnet | Gateway IP |
+| ---- | ------ | ---------- |
+| `management` | 10.201.0.0/24 | 10.201.0.1 |
+| `access`     | 10.202.0.0/24 | 10.202.0.1 |
+| `core`       | 10.203.0.0/24 | 10.203.0.1 |
+| `ran`        | 10.204.0.0/24 | 10.204.0.1 |
+
+### IP Addresses
+
+The following IP addresses are used in this tutorial.
+
+| Name | IP Address | Purpose |
+| ---- | ---------- | ------- |
+| `juju-controller.mgmt` | 10.201.0.10 | Management address for Juju machine
+| `control-plane.mgmt` | 10.201.0.11 | Management address for control plane cluster machine |
+| `user-plane.mgmt` | 10.201.0.12 | Management address for user plane cluster machine |
+| `api.juju-controller.mgmt` | 10.201.0.101 | Juju controller address |
+| `cos.mgmt` | 10.201.0.102 | Canonical Observability Stack address |
+| `upf.mgmt` | 10.201.0.151 | IP address for UPF to bind |
+| `amf.mgmt` | 10.201.0.201 | Externally reachable control plane endpoint for the AMF |
+
+The tutorial has been tested with a variety of users. We make every effort to keep it up-to-date and ensure that it’s reliable - but if you encounter any problems, we want to help you, so please let us know.
```

Comment:
Should we mention preferred  contact forms?

Reply:
Do we have one yet?

Reply:
We do already have this statement at the Tutorials section root level index, I don't think we need a second one here.